android/pytorch_android/src/androidTest/cpp/pytorch_jni_common_test.cpp

```diff
@@ -1,4 +1,7 @@
-// (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+//
+// This source code is licensed under the MIT license found in the
```

Comment:
PyTorch isn't MIT, so if you're going to have this in any headers, make sure it says BSD. Internal template looks like this:

```suggestion
// This source code is licensed under the BSD-style license found in the
```